prover/src/pp.rs

```diff
@@ -76,23 +88,39 @@ pub fn gen_to_file(k: usize, par: bool, pp_file: &str) -> Result<(), ProofError>
     let _guard = term_ctx.display_step();
 
     if par {
-        let pp: ParPP = gen_vm_pp(k)?;
-        show_pp(&pp);
-        save_pp(pp, pp_file)
+        if com {
+            let srs_file = srs_file.ok_or(ProofError::MissingSRS)?;
+            println!("Loading SRS from {srs_file}...");
```

Comment:
`nexus-tui` reserves the terminal window as a canvas and renders on it every 100ms. You will not see these prints.